### ISSUE TITLE
Report unsupported CRSs

### ISSUE THREAD

**Issue by jonherrmann** [etf-validator/etf-gmlgeox#22]
Currently unsupported CRSs are silently ignored.

Solution could be to call getReferencedObject **once for each CRS**:
```
CRSManager.getCRSRef(defaultSrsName).getReferencedObject()
```

**Comment by jonherrmann** [comment]
@jechterhoff I hope that reflects the observations from your analysis

**Comment by jechterhoff** [comment]
@jonherrmann Yes, it does. And good point about checking this only once per CRS.
